sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -236,6 +260,8 @@ case class CurrentBatchTimestamp(
     val timestampUs = millisToMicros(timestampMs)
     dataType match {
       case _: TimestampType => Literal(timestampUs, TimestampType)
+      case _: TimestampNTZType =>
+        Literal(convertTz(timestampUs, zoneId, ZoneOffset.UTC), TimestampNTZType)
```

Comment:
Should it be `convertTz(ts, ZoneOffset.UTC, zoneId)`?

Reply:
Yes. I make a mistake.